legacy-usage.md

```diff
@@ -0,0 +1,448 @@
+                    SignalFxProtocolBuffers.Property.newBuilder()
+                                    .setKey("version")
+                                    .setValue(
+                                            SignalFxProtocolBuffers.PropertyValue.newBuilder()
+                                                    .setIntValue(2)
+                                                    .build())
+                                    .build())
+                  .build());
+}
+```
+
+### Sending metrics through a HTTP proxy
+
+To send metrics through a HTTP proxy one can set the standard java system
+properties used to control HTTP protocol handling. There are 3 properties you
+can set to specify the proxy that will be used by the HTTP protocol handler:
+
+* `http.proxyHost`: the host name of the proxy server
+* `http.proxyPort`: the port number, the default value being 80.
+* `http.nonProxyHosts`: a list of hosts that should be reached directly, bypassing
+  the proxy. This is a list of regular expressions separated by `|`. Any host
+  matching one of these regular expressions will be reached through a direct
+  connection instead of through a proxy.
+
+Basic example:
+
+```
+$ java -Dhttp.proxyHost=webcache.mydomain.com -Dhttp.proxyPort=8080”
+```
+
+Example with directive to bypass proxy for `localhost` and `host.mydomain.com`:
+
+```
+$ java -Dhttp.proxyHost=webcache.mydomain.com -Dhttp.proxyPort=8080 -Dhttp.noProxyHosts=”localhost|host.mydomain.com”
+```
+
+### Disabling compression when sending datapoints
+
+By default, the Java library compresses datapoint payloads when sending them to
+SignalFx. This can provide significant egress volume savings when sending data
+to SignalFx's ingest API. This behavior can be disabled by setting the
+`com.signalfx.public.java.disableHttpCompression` system property to `true`:
+
+```
+$ java -Dcom.signalfx.public.java.disableHttpCompression=true ...
+```
+
+## Example Project
+
+You can find a full-stack example project called "signalfx-java-examples" in
+the repo.
+
+Run it as follows:
+
+1. Download the code and create an "auth" file in the "signalfx-java-examples"
+   directory. The auth file should contain the following:
+
+    ```
+    auth=<signalfx API Token>
+    host=https://ingest.signalfx.com
+    ```
+
+2. Run the following commands in your terminal to install and run the example
+   project, replacing `path/to/signalfx-java-examples` with the location of the
+   example project code in your environment. You must have Maven installed.
+
+    ```
+    cd path/to/signalfx-java-examples
+    mvn install
+    # an example for Yammer 2.x metrics
+    mvn exec:java -Dexec.mainClass="com.signalfx.example.YammerExample"
+    # an example for sending datapoints and events using protocol buffers
+    mvn exec:java -Dexec.mainClass="com.signalfx.example.ProtobufExample"
+    ```
+New metrics and events from the example project should appear in SignalFx.
+
+## Executing SignalFlow computations
```

Comment:
As much as I hate it, this is unfortunately not legacy usage; there's no replacement for this, and we should either keep this in the main readme or move it to some separate `signalflow.md` doc and link to it from the readme

Reply:
It links back to this repo though 😅

Reply:
I'll make it a task to edit it.